### DISCUSSION TITLE
Clear snapshots downloads

### DISCUSSION THREAD

**Discussion by Hr46ph** [mbnuqw/sidebery#1359] [Q&A]
After enabling timed export of snapshots, my Downloads list fills up. Is there a way to automatically clear these, or not show them there in the recent Downloads list?

**Comment by Hr46ph** [comment]
Anyone, please?

Not sure why it was closed so I reopened it.

**Comment by Heyian** [comment]
For anyone who is stumbling on this issue just like me, are on Windows and would like to automate the cleanup of the folder, I made a daily Task with the Task Scheduler to clean files older than 15 days in the folder. Here's how I did it :

1. Open Task Scheduler and make a new task, I named mine "Delete old Sideberry exports"

2. In the New Action window, specify the following data:

    Action: Start a program
    Program/script: `ForFiles.exe`
    Add arguments (optional): `/p "C:\Users\{user}\Downloads\Sidebery" /s /d -15 /c "cmd /c del @file"`
    Even though Windows says it's optional, it is not for our case, don't skip this.
    ### Don't forget to change the location to **_YOUR_** Sideberry export folder

3. Set the Trigger:
        Go to the Triggers tab and click on the New button.
        Under the Begin task, select a schedule in the drop-down list, I chose Daily.

4. Configure Settings:
        Switch to the Settings tab.
        Enable "Allow task to be run on demand" and "Run task as soon as possible after a scheduled start is missed".
        Click OK to create your task.

And that's it, it nows cleans up the folder everyday, it's super fast and I don't expect much hiccup with this method.

Don't forget to test it before simply letting it do it's job ;)
